crates/uv/src/commands/python/install.rs

```diff
@@ -407,14 +471,46 @@ pub(crate) async fn install(
         }
     }
 
+    // Read all existing managed installations and find the highest installed patch
+    // for each installed minor version.
+    let installations = ManagedPythonInstallations::from_settings(install_dir)?.init()?;
+    let existing_installations: Vec<_> = installations.find_all()?.collect();
+    let minor_versions =
+        PythonInstallationMinorVersionKey::highest_installations_by_minor_version_key(
+            existing_installations,
+        );
+
+    for installation in minor_versions.values() {
```

Comment:
Ensure the minor version links exist for the highest patch installations per minor version key.